### PR TITLE
UIF-473: add list of donors component

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UIF-473](https://issues.folio.org/browse/UIF-473) - add the list of donors
## Approach

https://github.com/folio-org/ui-finance/assets/116072773/632b9f0c-e6ea-42ef-b1c9-08c69317c636

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Funds/FundForm/FundForm.js [hunk 1]
```diff
@@ -410,6 +414,16 @@ const FundForm = ({
                         </Col>
                       </Row>
                     </Accordion>
+                    <Accordion
+                      closedByDefault
+                      id={SECTIONS_FUND.DONOR_INFORMATION}
+                      label={<FormattedMessage id="ui-finance.fund.information.donorInformation" />}
+                    >
+                      <DonorsList
```
Comment (usavkov-epam) [inline src/Funds/FundForm/FundForm.js:422]: "Add donor" btn lives in `<DonorsList>` components? It seems this point was missed during the review of the https://github.com/folio-org/stripes-acq-components/pull/724
As for me, those should be separate components.

Comment (alisher-epam) [reply src/Funds/FundForm/FundForm.js:422]: @usavkov-epam I am thinking of making it configurable through props.

**Review by BogdanDenis — APPROVED** [review]

File: src/Funds/FundForm/FundForm.js [hunk 2]
```diff
@@ -410,6 +414,18 @@ const FundForm = ({
...
+                    <Accordion
+                      closedByDefault
+                      id={SECTIONS_FUND.DONOR_INFORMATION}
+                      label={<FormattedMessage id="ui-finance.fund.information.donorInformation" />}
+                    >
+                      <Donors
+                        name="fund.donorOrganizationIds"
+                        donorOrganizationIds={donorOrganizationIds}
+                        visibleColumns={['name', 'code', 'unassignDonor']}
+                        columnWidths={{ name: '45%', code: '45%', unassignDonor: '10%' }}
```
Comment (usavkov-epam) [inline src/Funds/FundForm/FundForm.js:426]: let's omit setting the width here

File: src/Funds/FundForm/FundForm.js [hunk 3]
```diff
@@ -410,6 +414,18 @@ const FundForm = ({
...
                     </Accordion>
+                    <Accordion
+                      closedByDefault
+                      id={SECTIONS_FUND.DONOR_INFORMATION}
+                      label={<FormattedMessage id="ui-finance.fund.information.donorInformation" />}
+                    >
+                      <Donors
+                        name="fund.donorOrganizationIds"
+                        donorOrganizationIds={donorOrganizationIds}
+                        visibleColumns={['name', 'code', 'unassignDonor']}
```
Comment (usavkov-epam) [inline src/Funds/FundForm/FundForm.js:425]: hmm, I thought it was the default for the form component in acq-components

File: src/Funds/FundForm/FundForm.js (hunk 2 above)
Comment (alisher-epam) [reply src/Funds/FundForm/FundForm.js:426]: It doesn't look good without width

**Review by usavkov-epam — APPROVED** [review]

**Pull request merged**